LICENSE

```diff
@@ -0,0 +1,24 @@
+Copyright (c) 2018, The Pony Developers
```

Comment:
Copyright is wrong, should just be Pony Developers 2020.

I missed that when copying over from changelog-tool and tweaking.

Reply:
already fixed in 248fd5e